### PR TITLE
Include error message in TokenRefreshError

### PR DESCRIPTION
**Pull request by tsujigiri**

### CODE REVIEW

File: spec/soapy_bing/oauth_credentials_spec.rb [hunk 1]
```diff
@@ -83,28 +83,37 @@
...
   describe '#access_token' do
     let(:credentials) { { client_id: 'foo', client_secret: 'bar', refresh_token: 'baz' } }
-    let(:response) { double(:response) } # rubocop:disable RSpec/VerifiedDoubles
+    let(:response) do
+      HTTParty::Response.new(
+        OpenStruct.new(options: {}),
+        OpenStruct.new(code: status_code),
+        -> { response_body }
+      )
+    end
```
Comment (tsujigiri) [inline spec/soapy_bing/oauth_credentials_spec.rb:92]: I wanted to use `instance_double`, but it told me that `HTTParty::Response` doesn't implement the method `[]` for some reason, so I couldn't use it.

**Comment by tsujigiri** [comment]
[[BUG] Bing import broken in DWH](https://trello.com/c/ArDnaRbY/2451-bug-bing-import-broken-in-dwh)

File: spec/soapy_bing/oauth_credentials_spec.rb [hunk 2]
```diff
@@ -83,28 +83,37 @@
...
     before do
-      allow(response).to receive(:code).once.and_return(status_code)
       allow(HTTParty).to receive(:post).once.and_return(response)
     end
 
     context 'when there is good response' do
       let(:status_code) { 200 }
+      let(:response_body) { { 'access_token' => 'my-token' } }
 
       it 'memoizes http request response' do
```
Comment (itsDustin) [inline spec/soapy_bing/oauth_credentials_spec.rb:102]: typo here. "memoizes" => "memorizes"

**Comment by itsDustin** [comment]
🌊

File: spec/soapy_bing/oauth_credentials_spec.rb (hunk 2 above)
Comment (tsujigiri) [reply spec/soapy_bing/oauth_credentials_spec.rb:102]: https://en.wikipedia.org/wiki/Memoization

**Pull request merged**